internal/uvm/create_lcow.go

```diff
@@ -437,14 +438,16 @@ func makeLCOWVMGSDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_
 		},
 	}
 
+	maps.Copy(doc.VirtualMachine.Devices.HvSocket.HvSocketConfig.ServiceTable, opts.AdditionalHyperVConfig)
+
```

Comment:
Is there a reason we only add this for the Confidential case for LCOW?

Reply:
we do the same for normal LCOW tho:
https://github.com/microsoft/hcsshim/blob/da6f068ad62ebe3cb1da97ba69966fbbb28679e4/internal/uvm/create_lcow.go#L652
or am i misunderstanding your question?